### PR TITLE
Tab bar styling adjustment

### PR DESCRIPTION
**Pull request by caitlynjin**
## Overview

Minor adjustment with the tab bar design for the gym detail view.

## Changes Made

- Modified the `SlidingTabBarView` config:
    - Added bold attribute to both the selected and unselected tabs.
    - Set lighter text color for the unselected tab.
- Added a divider line at the bottom of the tab bar view.

## Screenshots

<details>
  <summary>Tab Bar</summary>
  <table>
    <tr>
      <td>Before</td>
      <td>After</td>
    </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
 </table>
</details>

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  </tr>
+ </table>
+</details>
+
+<!-- use the following if there is no before/after -->
+<details>
+  <summary>TODO</summary>
+  <img src="enterurlhere" width="300px" height="auto">
+  <video src="enterurlhere" type="video/mp4">
+</details>
```
Comment (vinnie4k) [inline .github/pull_request_template.md:66]: Nice template!

File: Uplift/Views/Supporting/SlidingTabBarView.swift [hunk 2]
```diff
@@ -68,10 +68,10 @@ extension SlidingTabBarView {
 
     /// Configuration for a tab bar.
     struct TabBarConfig {
-        var color: Color = Constants.Colors.black
-        var selectedFont: Font = Constants.Fonts.labelBold
+        var font: Font = Constants.Fonts.labelBold
+        var selectedColor: Color = Constants.Colors.black
```
Comment (vinnie4k) [inline Uplift/Views/Supporting/SlidingTabBarView.swift:72]: We should rename this to `selectedFontColor` to make it more clear and keep it parallel with the "Underline"

File: Uplift/Views/Supporting/SlidingTabBarView.swift [hunk 3]
```diff
@@ -68,10 +68,10 @@ extension SlidingTabBarView {
...
         var selectedUnderlineColor: Color = Constants.Colors.yellow
-        var unselectedFont: Font = Constants.Fonts.labelMedium
+        var unselectedColor: Color = Constants.Colors.gray04
```
Comment (vinnie4k) [inline Uplift/Views/Supporting/SlidingTabBarView.swift:74]: Same as above with the naming.

**Review by vinnie4k — APPROVED** [review]
Good to go. Just fix the naming and you're all set.

**Comment by belle-hu** [comment]
looks good!

**Review by belle-hu — APPROVED** [review]

**Review by vinnie4k — APPROVED** [review]
Good to go. Same thing with the other PR, you may or may not have to rebase with `main` after the other branches get merged.

**Pull request merged**